contracts/modules/capital/SwapOperator.sol

```diff
@@ -36,7 +40,9 @@ contract SwapOperator is ReentrancyGuard {
   IUniswapV2Router02 constant public router = IUniswapV2Router02(0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D);
   uint constant public MAX_LIQUIDITY_RATIO = 0.015 ether;
   uint constant public MIN_TIME_BETWEEN_SWAPS = 10 minutes;
-
+  address constant public enzymeV4VaultProxyAddress = 0x27F23c710dD3d878FE9393d93465FeD1302f2EbD;
```

Comment:
How likely is this address to change? 

And how would an asset migration look like to a new version.

Reply:
This should technically not change as it is immutable, however you should consider the possibility of you changing from one vault to another different vault with different strategies.